client/src/components/teams/TeamDashboard.vue

```diff
@@ -129,8 +129,8 @@
       </v-container>
       <v-divider />
 
-      <!-- Leaderboard -->
-      <v-row
+      <!-- Leaderboard: Hopefully coming soon!!  -->
```

Comment:
If we're leaving this for future devs then the comment should say that, otherwise it reads like we ran out of time